### PR TITLE
Satish/th yaml fix

### PR DESCRIPTION
**Pull request by satkarra**
Fixed bugs with TDyRegressionOutput that were introduced while extending to TH.
Added transiet-th-mpfao that was not in the travis yml file and the test makefile.
Commented PetscFree calls to tdy->Kappa and tdy->Kappa0 that were mainly causing the memory crashes.

### CODE REVIEW

File: src/tdyregression.c [hunk 1]
```diff
@@ -105,7 +129,12 @@ PetscErrorCode TDyRegressionInitialize(TDy tdy) {
   }
     
   ierr = ISCreateGeneral(PetscObjectComm((PetscObject)dm),global_count,int_array,PETSC_COPY_VALUES,&temp_is); CHKERRQ(ierr);
+
+//  ierr = ISView(temp_is,PETSC_VIEWER_STDOUT_WORLD); CHKERRQ(ierr);
```
Comment (bishtgautam) [inline src/tdyregression.c:133]: Please delete these commented out lines

File: src/tdyregression.c [hunk 2]
```diff
@@ -133,18 +164,76 @@ PetscErrorCode TDyRegressionOutput(TDy tdy, Vec U) {
...
+
+  if (tdy->mode == TH) {
+    ierr = VecMax(U_temp,NULL,&max_temp_val); CHKERRQ(ierr);
+    ierr = VecMin(U_temp,NULL,&min_temp_val); CHKERRQ(ierr);
+    ierr = VecSum(U_temp,&mean_temp_val); CHKERRQ(ierr);
+    ierr = VecGetSize(U_temp,&temp_vec_size); CHKERRQ(ierr);
+    mean_temp_val= mean_temp_val/temp_vec_size;
+  }
+
+//  ierr = VecView(U_pres,PETSC_VIEWER_STDOUT_WORLD); CHKERRQ(ierr);
```
Comment (bishtgautam) [inline src/tdyregression.c:235]: Please delete these commented out lines

File: src/tdyregression.c [hunk 3]
```diff
@@ -133,18 +164,76 @@ PetscErrorCode TDyRegressionOutput(TDy tdy, Vec U) {
   ierr = VecScatterBegin(reg->scatter_cells_per_process_gtos,U,reg->cells_per_process_vec,INSERT_VALUES,SCATTER_FORWARD);CHKERRQ(ierr);
   ierr = VecScatterEnd(reg->scatter_cells_per_process_gtos,U,reg->cells_per_process_vec,INSERT_VALUES,SCATTER_FORWARD);CHKERRQ(ierr);
 
+//  ierr = VecView(U,PETSC_VIEWER_STDOUT_WORLD); CHKERRQ(ierr);
```
Comment (bishtgautam) [inline src/tdyregression.c:167]: Please delete this commented out line

File: src/tdyregression.c [hunk 4]
```diff
@@ -81,12 +102,15 @@ PetscErrorCode TDyRegressionInitialize(TDy tdy) {
...
+
   for (c=0; c<global_count; c++){
     int_array[c] = c;
   }
-
+  
   ierr = ISCreateGeneral(PetscObjectComm((PetscObject)dm),global_count,int_array,PETSC_COPY_VALUES,&temp_is); CHKERRQ(ierr);
+
   ierr = VecScatterCreate(temp_vec,temp_is,regression->cells_per_process_vec,NULL,&temp_scatter); CHKERRQ(ierr);
+//  ierr = VecScatterView(temp_scatter,PETSC_VIEWER_STDOUT_WORLD); CHKERRQ(ierr);
```
Comment (bishtgautam) [inline src/tdyregression.c:113]: Please delete this commented out line

File: src/tdyregression.c (hunk 1 above)
Comment (satkarra) [reply src/tdyregression.c]: Done

File: src/tdyregression.c (hunk 3 above)
Comment (satkarra) [reply src/tdyregression.c]: Done

File: src/tdyregression.c (hunk 4 above)
Comment (satkarra) [reply src/tdyregression.c]: Done

File: src/tdyregression.c (hunk 2 above)
Comment (satkarra) [reply src/tdyregression.c]: Done

**Pull request merged**